pkgs/development/libraries/cpp-utilities/default.nix

```diff
@@ -1,14 +1,14 @@
-{ stdenv, fetchFromGitHub, cmake, cppunit }:
+{ gcc9Stdenv, fetchFromGitHub, cmake, cppunit }:
```

Comment:
Is gcc9Stdenv really needed? (I think the custom stdenvs are mostly for use outside of nixpkgs, to keep things simpler inside nixpkgs.)